dabest/_classes.py

```diff
@@ -1343,44 +1369,82 @@ def __pre_calc(self):
         out = []
         reprs = []
 
-        for j, current_tuple in enumerate(idx):
+        if not self.__is_preceding:
```

Comment:
Please change `preceding` to `repeated_measures` and `is_preceding` to ` repeated_measures`